server/routes/notifications.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+
+const { 
+    createNotification,
+    notificationList,
+    updateReadStatus,
+} = require("../controllers/notifications");
+
+router.route("/create").post(protect, createNotification);
+router.route("/all").get(protect, notificationList);
+router.route("/read").patch(protect, updateReadStatus);
```

Comment:
Great using `patch` here since it's a partial update and not updating the whole fields.